### DISCUSSION TITLE
请问一下form和form-item的使用方法，我安照官网的例子来报错了。。。

### DISCUSSION THREAD

**Discussion by lululuting** [TaleLin/lin-ui#1364] [求助] [answered]
<img width="1206" alt="WX20210406-193251@2x" src="https://user-images.githubusercontent.com/26959652/113706890-c200c280-9711-11eb-91fe-f13cf02d25f3.png">

**Comment by juzi214032** [comment, marked as answer]
贴一下代码呢

**Reply by lululuting** [reply]
就官网的基础例子

**Reply by juzi214032** [reply]
官网的示例是没有问题的

**Reply by ctt17** [reply]
我和你一样的情况

**Comment by lululuting** [comment]
那帮忙看一下怎么回事
![image](https://user-images.githubusercontent.com/26959652/113710507-62f17c80-9716-11eb-87c2-ac1992e012e9.png)
![image](https://user-images.githubusercontent.com/26959652/113710611-83b9d200-9716-11eb-8f59-966398b69bfb.png)
![image](https://user-images.githubusercontent.com/26959652/113710644-8d433a00-9716-11eb-9244-5c4c5be7198a.png)
![image](https://user-images.githubusercontent.com/26959652/113710665-959b7500-9716-11eb-9248-1675fdc85ca9.png)

**Reply by juzi214032** [reply]
发个代码片段看一下呢，从代码中我没看出来有什么问题
